src/pages/Checkout/index.jsx

```diff
@@ -0,0 +1,241 @@
+    const value = e.target.value;
+
+    const tmp = {...form};
+    tmp[name] = value;
+    setForm(tmp);
+  }
+
+  const handleForm = (e) => {
+    e.preventDefault();
+    console.log('Submit form',e);
+  }
+
+  return(
+    <Container>
+      <h2 style={{'textAlign':'center'}}>Customer information</h2>
+      <DividerSpace margin='1rem 0'/>
+      <Form id='myform' onSubmit={handleForm}>
+          <Input
+            value={form.name}
+            cb={handleInput}
+            label='Name'
+            type='text'
+            name='name'
+            placeholder='Please write your name ...'
+            maxLength="50"
+          />
+          <Input
+            value={form.email}
+            cb={handleInput}
+            label='Email'
+            type='email'
+            name='email'
+            placeholder='Please write your email ...'
+            maxLength="50"
+            pattern="\b[\w\.-]+@[\w\.-]+\.\w{2,4}\b"
+          />
+          <Input
+            value={form.pc}
+            cb={handleInput}
+            label='Postal Code'
+            type='text'
+            name='pc'
+            placeholder='Please write your postal code ...'
+            maxLength="5"
+          />
+          <Input
+            value={form.textarea}
+            cb={handleInput}
+            label='Order notes'
+            type='textarea'
+            name='textarea'
+            placeholder='Please write your order notes ...'
+            maxLength="250"
+          />
+          
+      </Form>
+      <Summary>
+        <h2 style={{'textAlign':'center'}}>Order summary</h2>
+        <div>
+          <Table>
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>No. products</th>
+                <th>Subtotal</th>
+              </tr>
+            </thead>
+            <tbody>
+              {
+                productsInCart.length > 0 &&
+                productsInCart.map(item=>{
+                  return(
+                    <tr key={item.product.id}>
+                      <td>{item.product.data.name}</td>
+                      <td>{item.amount}</td>
+                      <td>${item.product.data.price}</td>
+                    </tr>
+                  );
+                })
+              }
+            </tbody>
+            <tfoot>
+              <tr>
+                <td> </td>
+                <td><span>Total:</span></td>
+                <td>${productsInCart.reduce((valorAnterior,valorActual)=>{
+                        return valorAnterior+(valorActual.amount * valorActual.product.data.price);
+                      },0)}</td>
+              </tr>
+            </tfoot>
+          </Table>
+
+        </div>
+        <SummaryButtons>
+          <button type='submit' form='myform'>Place order</button>
+          <Link to='/cart'>
+            <button>Go back to cart</button>
+          </Link>
+        </SummaryButtons>
+      </Summary>
+
+    </Container>
+  );
+};
+
+export default Checkout;
```

Comment:
Larger react components can be rewritten in smaller components to achieved modularity and maintenance :)